docs/architecture/PROCESS.md

```diff
@@ -0,0 +1,56 @@
+# ADR Creation Process
+
+1. Copy the `adr-template.md` file. Use the following filename pattern: `adr-next_number-title.md`
+2. Create a draft Pull Request if you want to get an early feedback.
```

Comment:
2. Create a draft Pull Request and solicit input from the stewarding team, if you want to get early feedback.